Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -2210,6 +2225,18 @@ def lhegensim2018ml(fragment,howMuch):
                           },dataReco])
 
 steps['RECOCOSDRUN2']=merge([{'--conditions':'auto:run2_data','--era':'Run2_2016'},steps['RECOCOSD']])
+
+#Run 3
+steps['RECODR3']=merge([{'--scenario':'pp',
+                         '-s':'RAW2DIGI,L1Reco,RECO,DQM',
```

Comment:
PAT got lost here as well
it seems like this step is not used in any workflow though. so, no harm so far.